src/GoTrueClient.ts

```diff
@@ -1432,12 +1427,11 @@ export default class GoTrueClient {
     if (options?.scopes) {
       urlParams.push(`scopes=${encodeURIComponent(options.scopes)}`)
     }
-    if (options?.flowType === 'pkce') {
+    if (this.flowType === 'pkce') {
       const codeVerifier = generatePKCEVerifier()
       await setItemAsync(this.storage, `${this.storageKey}-code-verifier`, codeVerifier)
       const codeChallenge = await generatePKCEChallenge(codeVerifier)
       const flowParams = new URLSearchParams({
-        flow_type: `${encodeURIComponent(options.flowType)}`,
```

Comment:
Do we not still need to send the flowtype to the api?

Reply:
yeah, it's no longer necessary since https://github.com/supabase/gotrue/pull/1027